macros/quote_something.sql

```diff
@@ -0,0 +1,22 @@
+{%- macro quote(thing) -%}
```

Comment:
Not a critical comment, but I was not fully aware of what this macro was doing until I read your explanation in the README. Would a more appropriate name and argument be something along the lines of 
```sql
{%-database_quoting(db_name) -%}
```
I'm not entirely strong on the opinion that this should be renamed, but curious what your thoughts are.

Reply:
well the thing is _could_ be a database name, cut it could also be a column or table (or anything else you'd need to quote in SQL?) ... perhaps `sql_object` would be a better argument name? 

as for the macro name, i felt like simpler felt nice (and we don't add `db_*` to our other cross-database macros) but maybe something more descriptive like `wrap_in_quotes` would be better?

Reply:
Good points! I really like the `wrap_in_quotes` suggestion.

Reply:
cool updated the name of the macro (but left `thing` as the argument?)

Reply:
renamed `thing` to `object_to_quote`